### PR TITLE
Complete Project.

### PR DESCRIPTION
**Pull request by cod3smith**
### What the PR Contains?
- This project has only one feature. We worked with visual hierarchy. Overall web design is inspired by Smashing Magazine website design.

### Contributors.
- Sajad Ahmad.
- Kelyn Njeri

### CODE REVIEW

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hey @KelynPNjeri @SajjadAhmad14  :wave:
-
You have done a splendid job! :tada: Keep it up.
We, however, do have some few changes to carry out before we proceed (see review comment).
##
- [x] Good job with the summary. We can always improve on it though by highlighting what we have carried out in the project's PR to guide the reviewer on what has been achieved so far. We just have to pay attention to what;s being communicated and adhere to these guidelines of having a meaningful title and a short description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).
- [x] In the footer section, kindly implement the centred links (content is not important) ([link to page](https://www.smashingmagazine.com/)).

- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:
##
- When you have made the specified changes, please submit another code review. Thank you!
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+
+.article-img {
+  display: grid;
+  grid-template-columns: 1fr 3fr;
+  margin-bottom: 5px;
+}
+
+.img-2 {
+  /* position: absolute;
+  right: 30px; */
```
Comment (peterrobert) [inline assets/css/style.css]: Kindly remove the commented code. It will make your code clean

File: assets/css/style.css [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+  /* border: 3px solid green; */
+  width: 100%;
+  height: 700px;
+
+  /* background-color: silver; */
+  margin-bottom: 5px;
+}
+
+#img {
+  /* border: 3px solid red; */
```
Comment (peterrobert) [inline assets/css/style.css]: remove commented code.

File: assets/css/style.css [hunk 3]
```diff
@@ -0,0 +1,274 @@
+:root {
+  /* Shades of Grey to Highlight hierarchy. */
```
Comment (peterrobert) [inline assets/css/style.css]: remove commented code

File: assets/css/style.css [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+.medium {
+  background-color: var(--medium-hierarchy);
+}
+
+.low {
+  background-color: var(--low-hierarchy);
+}
+
+.nav-container {
+  /* background-color: grey; */
```
Comment (peterrobert) [inline assets/css/style.css]: remove commented code

File: assets/css/style.css [hunk 5]
```diff
@@ -0,0 +1,274 @@
...
+  font-weight: 900px;
+  text-align: center;
+}
+
+.items a:hover {
+  background-color: rgb(85, 81, 81);
+}
+
+.article2 {
+  /* background-color: rgb(192, 192, 192); */
```
Comment (peterrobert) [inline assets/css/style.css]: remove commented code.

File: assets/css/style.css [hunk 6]
```diff
@@ -0,0 +1,274 @@
...
+
+.article2 .articles {
+  display: grid;
+  align-content: center;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.background-container {
+  /* border: 3px solid green; */
```
Comment (peterrobert) [inline assets/css/style.css]: remove commented code

File: assets/css/style.css [hunk 7]
```diff
@@ -0,0 +1,274 @@
...
+  width: 100%;
+  height: 700px;
+
+  /* background-color: silver; */
```
Comment (peterrobert) [inline assets/css/style.css]: remove commented code

**Comment by peterrobert** [comment]
**PULL REQUEST UNDER REVIEW BY PETER ROBERT**

**Review by peterrobert — CHANGES_REQUESTED** [review]
**CHANGE REQUIRED**:stop_sign: 
Good work so far :clap:  Great work implementing the changes:+1:  However there are a few issues that require change before I could approve this project.

-  Remove all the commented code. This will keep the code clean.

-  There are some issues in your CSS file. Kindly fix them

css

**Review by HADDADSOHAIB — APPROVED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
## Approved :+1: 

### Hello @KelynPNjeri  and @SajjadAhmad14    :handshake: 

### Good job so far :clap:  your project is looking good :medal_sports: 

### You implemented all the required changes by the previous TSEs so the project is approved :point_up:

**Pull request merged**